### PR TITLE
Make use_tidy_github_actions() more pushy

### LINKED ISSUE #1367: use_tidy_github_actions() should be more forceful
I don't think it needs to prompt to overwrite because (a) we know that the user must be using git + github and (b) the user are mostly us. Should also avoid the repeated advice to see the readme.

### PR DESCRIPTION
**Pull request by hadley**
Since we (presumably) know what we're doing. Fixes #1367.

### CODE REVIEW

File: R/github-actions.R [hunk 1]
```diff
@@ -160,12 +161,12 @@ use_github_action <- function(name,
...
   create_directory(path_dir(proj_path(save_as)))
 
-  new <- write_over(proj_path(save_as), contents)
+  new <- write_over(proj_path(save_as), contents, overwrite = overwrite)
 
   if (open && new) {
     edit_file(proj_path(save_as))
   }
-  if (!is.null(readme)) {
+  if (!is.null(readme) && !overwrite) {
```
Comment (hadley) [inline R/github-actions.R:169]: This is not 100% correct but is expedient, and I don't think another argument is worth it.

**Review by jimhester — APPROVED** [review]

File: R/write.R [hunk 2]
```diff
@@ -69,16 +69,17 @@ write_union <- function(path, lines, quiet = FALSE) {
 #'   necessary or overwriting existing, if proposed contents are not identical
 #'   and user is available to give permission.
 #' @param contents Character vector of lines.
+#' @param overwrite Unconditionally an existing file.
```
Comment (jennybc) [inline R/write.R:72]: There seem to be some missing words here

**Review by jennybc — APPROVED** [review]
Feels like this suggests a tweak or addition to the `write_over()` tests

So: when `overwrite = FALSE`, user will be asked if they are OK with replacing the existing workflow?

I suspect we could be pushy more globally, i.e. whenever we see the project `uses_git()`.

**Comment by hadley** [comment]
Hmmmm, I like the idea of suppressing this question if you already use git. What do you think about just doing that globally?

**Comment by jennybc** [comment]
> Hmmmm, I like the idea of suppressing this question if you already use git. What do you think about just doing that globally?

I'm on board.

**Comment by hadley** [comment]
Closing in favour of adjusting `write_over()`

**Pull request closed without merging**